src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,73 @@
+import { MaterialDesignDescription, ImageGrid } from '../../app/components';
+import { PictureAsPdf as PDFIcon } from '@material-ui/icons';
+import LightThemeImage from '../../app/assets/design-patterns/illustrations/illustration-light-theme.svg';
+import DarkThemeImage from '../../app/assets/design-patterns/illustrations/illustration-dark-theme.svg';
+import BackgroundShapeImage from '../../app/assets/design-patterns/illustrations/illustration-background.png';
+
+<!-- KEYWORDS: design illustration graph onboarding -->
+
+# Illustrations
+
+Illustrations are design elements that should be used mostly for onboarding screens and [empty states](/patterns/empty-states). Do not use illustrations where you are supposed to use icons.
```

Comment:
```suggestion
Illustrations are bold graphics that are meant to stand out on a page. They should be used primarily for onboarding screens and [empty states](/patterns/empty-states). 

> Illustrations are meant to be large and detailed graphics. Do not use them in places where icons are more appropriate.
```
Will our empty state components support a png/svg instead of an icon?

Reply:
React takes `ReactNode` 
React Native takes `React.Component<{ size: number, color: string }>`

Reply:
Okay, React should be fine...RN will probably need an update - want to log an enhancement issue? And Angular is just content projection right? So it can be anything?

Reply:
Angular: correct. As long as it has `pxb-empty-icon` as the selector

Reply:
https://github.com/pxblue/react-native-component-library/issues/69